src/app/auth/login/login.page.ts

```diff
@@ -39,4 +61,21 @@ export class LoginPage implements OnInit {
     this.authService.logout();
   }
 
+    // validation message when input information is incorrect
+    validation_messages = {
```

Comment:
Please match the indentation to previous code block. Also move the declaration to before constructor.

Reply:
There should be automatic code formatting check if you use visual studio code. there is linter file in the project which looks after code formatting.

Reply:
Thanks for your comments and information. I will check and modify it.